source/api/extension/navplace/index.md

```diff
@@ -0,0 +1,331 @@
+                        "service":[
+                           {
+                              "id":"https://iiif.io/api/image/3.0/example/reference/918ecd18c2592080851777620de9bcb5-gottingen",
+                              "profile":"level1",
+                              "type":"ImageService3"
+                           }
+                        ]
+                     },
+                     "target":"https://example.org/iiif/canvas/p1"
+                  }
+               ]
+            }
+         ]
+      }
+   ],
+   "navPlace":{
+      "id" : "https://example.org/iiif/feature-collection/2",
+      "type":"FeatureCollection",
+      "features":[
+         {
+            "id":"https://example.org/iiif/feature/2",
+            "type":"Feature",
+            "properties":{
+               "label":{
+                  "en":[
+                     "navPlace Extension Example Manifest"
+                  ]
+               }
+            },
+            "geometry":{
+               "type":"Point",
+               "coordinates":[
+                  9.938,
+                  51.533
+               ]
+            }
+         }
+      ]
+   }
+}
+```
+
+## 5. Implementation Notes
+
+
+### 5.1 GeoJSON-LD on the Web
+
+The choice to use GeoJSON-LD is not arbitrary. It is the primary geographic coordinate data format supported by web-based map platforms. This extension drives implementers to have geographic coordinate data saved as GeoJSON-LD when it is intended for use on the web. This practice removes barriers for interoperable geographic coordinate data as well as promotes the robust functionality of prior, current, and upcoming web-based map platform developments.
+
+
+### 5.2 IIIF Cookbook
+
+The [IIIF Cookbook](https://iiif.io/api/cookbook/) exemplifies implementations of digital objects under IIIF Presentation API 3. IIIF Cookbook recipes exemplifying use of the `navPlace` property are under development. This specification will update to include links to those recipes as they are published.
+
+
+## Appendices
+
+### A. Acknowledgements
+
+An enormous amount of gratitude is owed to the IIIF community for their continuous engagement, innovative ideas, and feedback. Thanks to [Sean Gilles](https://github.com/sgillies) and MapBox for the GeoJSON linked data context and its promotion of standardized geographic web data. We would also like to recognize the [IETF](https://www.ietf.org/) for the semantics produced through the GeoJSON specification. An extra special thank you goes out to the [IIIF Maps](https://iiif.io/community/groups/maps/) and [IIIF Cookbook](https://iiif.io/api/cookbook/) communities as implementers of this developing technology. The initial version of this document was the work of the [IIIF Maps Technical Specification Group](https://iiif.io/community/groups/maps-tsg/).
+{: #acknowledgements}
+
+
+### B. Change Log
+{: #change-log}
+
+| Date       | Description           |
+| ---------- | --------------------- |
```

Comment:
ChangeLog should be start with publication, no need for "Initial commit" or copying google docs.